controllers/microk8sconfig_controller.go

```diff
@@ -240,12 +240,6 @@ func (r *MicroK8sConfigReconciler) handleClusterNotInitialized(ctx context.Conte
 		return ctrl.Result{RequeueAfter: 30 * time.Second}, nil
 	}
 
-	// if the machine has not ClusterConfiguration and InitConfiguration, requeue
-	if scope.Config.Spec.InitConfiguration == nil && scope.Config.Spec.ClusterConfiguration == nil {
-		scope.Info("Control plane is not ready, requeueing joining control planes until ready.")
-		return ctrl.Result{RequeueAfter: 30 * time.Second}, nil
-	}
-
```

Comment:
IMO we can remove this from now. We can add a similar kind of logic back once the `ClusterConfiguration` and `InitConfiguration` are used in the future (for adding custom certificates, for instance)